.gitignore

```diff
@@ -1,48 +1,49 @@
-*.exe
```

Comment:
why'd you rearrange this file?

Reply:
i did a quick `:sort`, not for any practical reason.
it just looked messy...

Reply:
the issue is that the diff algorithm is freaking out, so if you added or deleted any lines, it isn't as obvious

Reply:
added `*.diff`